project.clj

```diff
@@ -5,7 +5,10 @@
             :url "https://www.apache.org/licenses/LICENSE-2.0"}
   :dependencies [[com.google.guava/guava "15.0"] ; Explicitly pull new Guava version for dependency conflicts.
                  [org.clojure/clojure "1.7.0"]
-                 [com.cerner/clara-rules "0.18.0"]
+                 [com.cerner/clara-rules "0.19.1"]
+
+                 ;; Dependencies for com.cemerick/austin
+                 [org.clojure/tools.nrepl "0.2.13"]
```

Comment:
This is a dep of `com.cemerick/austin` which is a pretty old thing to use for cljs. 
If this is accurate, I think that:
1) It'd make more sense for it to be in the `:dev` profile of `:profiles`
2) As is the case in the main clara project, I think the cljs hooks should be removed so that the clj-side repl setup doesn't require cljs building/running at all. Some people don't care about that.

(2) Is probably something that can be looked at later once the main `clara-rules` project is doing a similar pattern.